INSTALL.md

```diff
@@ -2,24 +2,10 @@
 
 ## For the casual user
 
-If you want to write a simple script, don't care about a reproducible client
-library install, don't mind getting head (which may be less stable than a
-particular release) and don't share any dependencies with client-go, then
-simply:
-
-```sh
-$ go get k8s.io/client-go/...
-
-# Test install
-$ go build k8s.io/client-go/examples/...
-```
-
-This will install `k8s.io/client-go` in your `$GOPATH`. `k8s.io/client-go` includes its own
-dependencies in its `k8s.io/client-go/vendor` path. `go get` will not flatten
-those dependencies into your `$GOPATH`, which means they will be distinct from
-any dependencies you may already have there. This will be problematic if you
-happen to already have a copy of, say, glog, and in that case you'll need to
-look down at the next section.
+Currently, there is no super easy way to use client-go.  Hopefully this will
+change soon. Simply running `go get k8s.io/client-go/...` will leave you with a
+library that can't practically be used.  It is important to synchronize your
+dependencies with the ones that are required by the library.
```

Comment:
@caesarxuchao how long before your changes to remove the offending dependencies land?